Source/PTEDashboard.m

```diff
@@ -106,14 +106,13 @@ - (void)handleStatusBarOrientationChange:(NSNotification *)notification
     UIInterfaceOrientation nextOrientation = [notification.userInfo[UIApplicationStatusBarOrientationUserInfoKey] integerValue];
     
     // Flip the window's height and width?
-    CGRect frame = self.frame;
-    if ((UIDeviceOrientationIsLandscape(currentOrientation) && UIDeviceOrientationIsPortrait(nextOrientation)) ||
-        (UIDeviceOrientationIsPortrait(currentOrientation) && UIDeviceOrientationIsLandscape(nextOrientation)))
-    {
-        frame.size = CGSizeMake(frame.size.height,
-                                frame.size.width);
-    }
-    
+	CGRect frame = self.frame;
+	if ((UIInterfaceOrientationIsLandscape(currentOrientation) && UIInterfaceOrientationIsPortrait(nextOrientation)) ||
+		(UIInterfaceOrientationIsPortrait(currentOrientation) && UIInterfaceOrientationIsLandscape(nextOrientation)))
+	{
+		frame.size = CGSizeMake(frame.size.height,frame.size.width);
+	}
```

Comment:
Drag your mouse in your indentation code here and you'll notice that it indents with either tab for each level.

Regular Xcode indentation (so this and most iOS/OSX projects) uses four spaces for each level of indentation.

You can take a look here at how to change the setting and some discussion about indentation ;)

http://stackoverflow.com/a/3336065/1049134